app/helpers/application_helper.rb

```diff
@@ -30,8 +30,8 @@ def session_with_trust?
   end
 
   def page_with_trust?
-    current_page?(controller: 'sign_up/passwords', action: 'new') ||
-      current_page?(controller: 'users/reset_passwords', action: 'edit')
+    return false if current_page?(controller: 'users/sessions', action: 'new')
+    return true
```

Comment:
Inconsequential, but curious: Any reason not to change flatten this to a single return path?

```suggestion
    return !current_page?(controller: 'users/sessions', action: 'new')
```

Reply:
no reason other than I didn't think of it. I do like that version more.